package.json

```diff
@@ -2,22 +2,18 @@
   "name": "@lumigo/opentelemetry",
   "version": "1.19.1",
   "description": "Lumigo wrapper to trace distributed architecture",
-  "main": "dist/src/wrapper.js",
+  "main": "dist/src/distro.js",
   "private": false,
-  "types": "dist/src/wrapper.d.ts",
+  "types": "dist/src/bootstrap.d.ts",
   "scripts": {
-    "build": "rm -rf dist wrapper.tgz && tsc --build --force && tracer=\"$(npm pack)\" && mv \"$tracer\" wrapper.tgz",
+    "build": "rm -rf dist *.tgz && tsc --build --force && tracer=\"$(npm pack)\" && mv \"$tracer\" distro.tgz && cp distro.tgz wrapper.tgz",
```

Comment:
This is a bit of silliness not to break integration tests, that unfortunately have hard-coded the `wrapper.tgz` reference